pybaseball/batting_leaders.py

```diff
@@ -1,51 +1,34 @@
-_FG_BATTING_LEADERS_URL = "/leaders.aspx?pos=all&stats=bat&lg={league}&qual={qual}&type={types}&season={end_season}&month=0&season1={start_season}&ind={ind}&team=&rost=&age=&filter=&players=&page=1_100000"
-# TODO: update url to include more stats
+import warnings
+from typing import Optional
 
+import pandas as pd
 
-def batting_stats(start_season: int, end_season: int = None, league: str = 'all', qual: int = 1, ind: int = 1) -> pd.DataFrame:
-    if start_season is None:
-        raise ValueError(
-            "You need to provide at least one season to collect data for. Try batting_leaders(season) or batting_leaders(start_season, end_season)."
-        )
-    if end_season is None:
-        end_season = start_season
-
-    column_mapper = BattingStatsColumnMapper()
-
-    fg_data = fangraphs.get_fangraphs_tabular_data_from_url(
-        _FG_BATTING_LEADERS_URL.format(
-            league=league,
-            qual=qual,
-            types=_FG_BATTING_LEADERS_TYPES,
-            end_season=end_season,
-            start_season=start_season,
-            ind=ind
-        ),
-        column_name_mapper=column_mapper.map,
-        known_percentages = ['GB/FB']
+from pybaseball.datasources.fangraphs import FanGraphsLeague, fg_batting_data
+
+
+def batting_stats(start_season: int, end_season: int = None, league: str = 'all', qual: Optional[int] = None,
+                  ind: int = 1) -> pd.DataFrame:
+    """
+    Get season-level batting data from FanGraphs.
+
+    ARGUMENTS:
+    start_season : int           : first season you want data for
+                                   (or the only season if you do not specify an end_season)
+    end_season   : int           : final season you want data for
+    league       : str           : "all", "nl", or "al"
+    qual         : Optional[int] : minimum number of plate appearances to be included in the data.
+                                   If None is specified, the FanGraphs default ('y') is used.
+                                   Default = None
+    ind          : int           : 1 if you want individual season-level data
+                                   0 if you want a player's aggreagate data over all seasons in the query
+    """
+
+    warnings.warn("batting_stats is deprecated in favor of fg_batting_data", DeprecationWarning)
+
+    return fg_batting_data(
+        start_season,
+        end_season=end_season,
+        league=FanGraphsLeague(league),
+        qual=qual,
+        split_seasons=(ind=='1')

```

Comment:
if this is the same, why isn't `batting_stats` the same as `fg_batting_data`? why do we need separate hooks for this?